### ISSUE TITLE
Annotation/Reply body text areas accessibility improvements

### ISSUE THREAD

**Issue by acelaya** [hypothesis/client#6179]
> 1. Annotation body text areas lack a visible label
> 2. Annotation body text areas use the same label for annotations and replies

Rob's comment regarding point 1: "I'm not convinced that we should blindly add a label outside the text area, since I think the purpose is fairly obvious from the context. A placeholder might be adequate"
